src/Port.js

```diff
@@ -0,0 +1,12 @@
+function Port(name) {
+  this.name = name;
+  this.ships = [];
+}
+Port.prototype.addShip = function (ship) {
+  this.ships.push(ship);
+};
+Port.prototype.removeShip = function (ship) {
```

Comment:
A small formatting correction here, its common to leave a clear line between each method so its easy to see where one ends and another begins at a glance
```
}

Port.prototype.addShip = function (ship) {
  this.ships.push(ship);
};

Port.prototype.removeShip = function (ship) {
```